.github/workflows/lint.yaml

```
@@ -18,4 +18,4 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
```

You are bumping this below so maybe a good idea here? 
```suggestion
      - uses: actions/checkout@v4
```